From b571ca4e86ffacb69c6dfe5f38cec27e6c825498 Mon Sep 17 00:00:00 2001
From: Matthew Fernandez <matthew.fernandez@gmail.com>
Date: Fri, 3 Jul 2020 11:21:47 -0700
Subject: [PATCH] suppress compiler warning from qsort() call

---
 lib/common/ns.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/common/ns.c b/lib/common/ns.c
index 282f2ba52..4869272f7 100644
--- a/lib/common/ns.c
+++ b/lib/common/ns.c
@@ -706,7 +706,8 @@ static void TB_balance(void)
     }
     Tree_node.size = ii;
     qsort(Tree_node.list, Tree_node.size, sizeof(Tree_node.list[0]),
-        adj > 1? decreasingrankcmpf : increasingrankcmpf);
+        adj > 1? (int(*)(const void*,const void*))decreasingrankcmpf
+               : (int(*)(const void*,const void*))increasingrankcmpf);
     for (i = 0; i < Tree_node.size; i++) {
         n = Tree_node.list[i];
         if (ND_node_type(n) == NORMAL)
-- 
2.45.2

