--- graphviz-2.42.1.orig/configure.ac
+++ graphviz-2.42.1/configure.ac
@@ -2255,15 +2255,15 @@ AC_ARG_WITH(ann,
 if test "x$with_ann" != "xyes"; then
   use_ann="No (disabled)"
 else
-  PKG_CHECK_MODULES(ANN, [ann],[
-    use_ann="Yes"
+  if test -f /usr/lib/${host_cpu}-linux-gnu/libann.so; then
     AC_DEFINE_UNQUOTED(HAVE_ANN,1,
       [Define if you have the ann library])
-    AC_SUBST([ANN_CFLAGS])
+    ANN_LIBS=-lann
     AC_SUBST([ANN_LIBS])
-  ],[
-    use_ann="No (no ann.pc found)"
-  ])
+    use_ann="Yes (but no ann.pc found)"
+  else
+    use_ann="No (ANN library not available)"
+  fi
 #  if test "$use_ann" != "Yes"; then
 #    if test -f /usr/lib$LIBPOSTFIX/libANN.so; then
 #      AC_DEFINE_UNQUOTED(HAVE_ANN,1,
