Description: Add a missing prototype for makeTetrix().
Author: Andrey Rakhmatullin <wrar@debian.org>
Bug-Debian: https://bugs.debian.org/1066358
Origin: backport, https://gitlab.com/graphviz/graphviz/-/commit/be6f649995d00865e7c7d721f9b5bdb13fd715c0
Forwarded: not-needed
Last-Update: 2024-03-14

--- graphviz-2.42.2.orig/cmd/tools/graph_generator.h
+++ graphviz-2.42.2/cmd/tools/graph_generator.h
@@ -30,6 +30,7 @@ extern void makeRandom(int, int, edgefn)
 extern void makeSquareGrid(int, int, int, int, edgefn);
 extern void makeBinaryTree(int, edgefn);
 extern void makeSierpinski(int, edgefn);
+extern void makeTetrix(int, edgefn);
 extern void makeHypercube(int, edgefn);
 extern void makeTree(int, int, edgefn);
 extern void makeTriMesh(int, edgefn);
