Description: remove bashism from gvmap.sh
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Bug-Debian: https://bugs.debian.org/772234
Last-Update: 2016-06-10

---

--- graphviz-2.38.0.orig/cmd/gvmap/gvmap.sh
+++ graphviz-2.38.0/cmd/gvmap/gvmap.sh
@@ -64,7 +64,7 @@ do
     exit 2
     ;;
   \? )
-    if [[ "$OPTARG" == '?' ]]
+    if [ "$OPTARG" = '?' ]
     then
       getopts -a gvmap "$OPTSTR" x '-?'
       exit 0
@@ -76,19 +76,19 @@ do
 done
 shift $((OPTIND-1))
 
-if [[ $# == 0 ]]
+if [ $# -eq 0 ]
 then
-  if [[ -n $VERBOSE ]]
+  if [ -n $VERBOSE ]
   then
     print -u 2 "$LAYOUT -Goverlap=prism $FLAGS1 | gvmap $FLAGS2 | neato -n2 $FLAGS3"
   fi
   $LAYOUT -Goverlap=prism $FLAGS1 | gvmap $FLAGS2 | neato -n2 $FLAGS3
 else
-  while (( $# > 0 ))
+  while $(( $# > 0 ))
   do
-    if [[ -f $1 ]]
+    if [ -f $1 ]
     then
-      if [[ -n $VERBOSE ]]
+      if [ -n $VERBOSE ]
       then
         print -u 2 "$LAYOUT -Goverlap=prism $FLAGS1 $1 | gvmap $FLAGS2 | neato -n2 $FLAGS3"
       fi
